lightning/src/ln/onion_message.rs

```diff
@@ -0,0 +1,69 @@
+/// The payload of an inbound onion message.
+pub struct Payload {
```

Comment:
Maybe prefix names with `Inbound` ?

Reply:
Now this struct is for inbound and outbound :)